release_notes/41-release-notes.adoc

```diff
@@ -340,9 +345,26 @@ To address this change, you must update the configurations accordingly to ensure
 The `ROX_FORCE_LOCAL_IMAGE_SCANNING` environment variable has been removed in this release. To address this change, replace references to this variable with the one that uses the delegated registry configuration.
 
 [id="kernel-module-collection-method_{context}"]
-==== Kernel module collection method
+==== Kernel module collection method (updated 7 July 2023)
+
+The kernel module collection method has been removed in this release following its deprecation in {product-title-short} 4.0. To address this change, secured clusters that are configured to use kernel module collection are automatically switched to eBPF. However, the following known issue exists:
```

Comment:
```suggestion
The kernel module collection method has been removed in this release following its deprecation in {product-title-short} 4.0. The following known issue exists:
```

Reply:
I had left that part in because I thought some may have consulted the release notes when they were published, and would wonder why it was removed. But I think leaving it in may be more confusing, so I've made this edit - thanks!